<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>638</width>
                <height>422</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,7,0,0">
                        <item>
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>当前状态</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_6">
                                <property name="text">
                                    <string>在线</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="test">
                                <property name="text">
                                    <string>测试</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="sendRedisData">
                                <property name="text">
                                    <string>发送缓存数据</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
                        <item>
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>当前用户</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>当前用户</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QRadioButton" name="radioButton">
                        <property name="text">
                            <string>是否开启浏览器最大化</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>测试用例并行数</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="comboBox">
                                <item>
                                    <property name="text">
                                        <string>1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>5</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>6</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>7</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>8</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>9</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>10</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QTextEdit" name="textEdit"/>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>638</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menu_2">
                <property name="title">
                    <string>执行器设置</string>
                </property>
            </widget>
            <addaction name="menu_2"/>
        </widget>
        <action name="action">
            <property name="text">
                <string>页面对象</string>
            </property>
        </action>
        <action name="action_2">
            <property name="text">
                <string>测试用例</string>
            </property>
        </action>
        <action name="action_3">
            <property name="text">
                <string>场景用例组</string>
            </property>
        </action>
        <action name="action_4">
            <property name="text">
                <string>公共方法</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
